src/web/mod.rs

```diff
@@ -413,7 +413,9 @@ async fn apply_middleware(
     Ok(router.layer(
         ServiceBuilder::new()
             .layer(TraceLayer::new_for_http())
-            .layer(sentry_tower::NewSentryLayer::new_from_top())
+            // FIXME: send / sync error?
```

Comment:
I'm not sure why this breaks, my guess is that sentry-tower is the problem, because its `SentryLayer` contains `_hub: PhantomData<(H, Request)>`, where the axum request is not `Sync`. 

I might be wrong, but will have to do more digging on this. If I'm right then Crates.io will likely also run into this same issue, will also see how they fix it

Reply:
@Turbo87 you might run into the same issue on crates.io, my guess is that the solution will be the fix in sentry-tower: 

- https://github.com/getsentry/sentry-rust/issues/720
- https://github.com/getsentry/sentry-rust/compare/master...syphar:sentry-rust:no-phantom?expand=1

Reply:
Interesting. We're using the MatchedPath sentry extension for axum which is still using the old version, which is why I hadn't updated yet. I was assuming that that would be all that needs updating 😅

Reply:
I didn't realize sentry-tower implemented this :) 

( I [build a small middleware for the same thing](https://github.com/rust-lang/docs.rs/blob/144032ca545a2073c7b72053ead6d88322a14c6c/src/web/mod.rs#L385-L400)) 

> Interesting. We're using the MatchedPath sentry extension for axum which is still using the old version, which is why I hadn't updated yet. I was assuming that that would be all that needs updating 😅

My guess is that it will break the same way :)

Reply:
It did after I implemented it a while ago :D